packages/typegpu/src/shared/meta.ts

```diff
@@ -1,7 +1,14 @@
-import { $getNameForward } from './shared/symbols.ts';
+import type { ArgNames, Block } from 'tinyest';
+import { $getNameForward } from './symbols.ts';
 
-interface MetaData {
-  name: string | undefined;
+export interface MetaData {
```

Comment:
Could we also store versioning information in the `MetaData` type, to be injected by `unplugin-typegpu`?

```ts
interface MetaData {
  ast?: {
    v: number, // e.g. 11
    // ...
  } | undefined,
  // ...
}
```
The value in `ast.v` would have to be a constant exported from `tinyest` that can be used to differentiate between breaking changes in the format.